terraform/pas_terraform/modules/group/main.tf

```diff
@@ -0,0 +1,4 @@
+resource "rafay_group" "group" {
+  name        = var.group
+   description = "Project Admin"
```

Comment:
Change it to "group description" or leave it empty

Reply:
Done, removed description